.github/workflows/continuous-integration-workflow.yml

```diff
@@ -57,6 +57,12 @@ jobs:
         make -C build/ test
         python3 -m pytest
 
+    - name: Use pip to install speculos in a virtual environment
+      run: |
+        python3 -m venv venv-test
+        ./venv-test/bin/pip install .
```

Comment:
Installing speculos on Ubuntu 20.04 thanks to pip produces an error (but doesn't make pip install fail):

```shell
$ docker run --rm -it ubuntu:20.04
root@2c48a0145b49:/# apt-get -qy update
root@2c48a0145b49:/# apt-get install -qy cmake gcc gcc-arm-linux-gnueabihf git libc6-dev-armhf-cross libvncserver-dev make qemu-user-static wget
root@2c48a0145b49:/# apt install -qy python3-pip python3-venv
root@2c48a0145b49:/# python3 -m venv venv-test
root@2c48a0145b49:/# ./venv-test/bin/pip install git+https://github.com/LedgerHQ/speculos.git@add-setup-py
...
Building wheels for collected packages: speculos, construct
  Building wheel for speculos (setup.py) ... error
  ERROR: Command errored out with exit status 1:
   command: /speculos/venv-test/bin/python3 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-req-build-3kjx96mx/setup.py'"'"'; __file__='"'"'/tmp/pip-req-build-3kjx96mx/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /tmp/pip-wheel-jq88sw5r
       cwd: /tmp/pip-req-build-3kjx96mx/
  Complete output (6 lines):
  usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
     or: setup.py --help [cmd1 cmd2 ...]
     or: setup.py --help-commands
     or: setup.py cmd --help
  
  error: invalid command 'bdist_wheel'
  ----------------------------------------
  ERROR: Failed building wheel for speculos
  Running setup.py clean for speculos
  Building wheel for construct (setup.py) ... error
  ERROR: Command errored out with exit status 1:
   command: /speculos/venv-test/bin/python3 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-k2_6x3zr/construct/setup.py'"'"'; __file__='"'"'/tmp/pip-install-k2_6x3zr/construct/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /tmp/pip-wheel-kmz2_0b2
       cwd: /tmp/pip-install-k2_6x3zr/construct/
  Complete output (6 lines):
  usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
     or: setup.py --help [cmd1 cmd2 ...]
     or: setup.py --help-commands
     or: setup.py cmd --help
  
  error: invalid command 'bdist_wheel'
  ----------------------------------------
  ERROR: Failed building wheel for construct
...
```

This patch seems to fix this error:

```patch
$ git diff
diff --git a/setup.py b/setup.py
index c0bfe8c..2cb2adf 100755
--- a/setup.py
+++ b/setup.py
@@ -53,6 +53,7 @@ setup(
         "pyqt5>=5.15.2,<6.0.0",
         "requests>=2.25.1,<3.0.0",
     ],
+    setup_requires=['wheel'],
     entry_points={
         "console_scripts": [
             "speculos = speculos.main:main",
```

Reply:
I was a little bit reluctant first, as using `wheel` seems to be more an acceleration (to be able to download wheels from PyPI directly) than something required, but it seems that many packages do this, cf. https://grep.app/search?q=setup_requires%3D%5B%27wheel%27%5D%2C

So I added your suggestion and tested that it worked. Thanks!